client/my-sites/pages/page/index.js

```diff
@@ -50,10 +48,17 @@ function recordEvent( eventAction ) {
 
 // FIXME(mcsf): I vow to follow-up on this and port to Component
 const Page = React.createClass( { // eslint-disable-line react/prefer-es6-class
```

Comment:
Perhaps remove the disable line, since we do want to clean this up later and the tests are already red.